remoteit/desktop
# DESK-947 device details failure

### Description


Account for device not in currently selected device list on device details page

### Test plan



1. Sign in
2. Go to another shared device list
3. Click settings and then your own device
4. Navigate to the device details
5. You should see the device details

### Ticket



<https://remoteit.atlassian.net/browse/DESK-947>
